src/appsettings/settings.py

```diff
@@ -837,6 +898,20 @@ def __init__(
         if max_length is not None:
             self.validators.append(MaxLengthValidator(max_length))
 
+    def decode_environ(self, value):
+        """
+        Split value by colons to a list.
+
+        Args:
+            value (str):
+
+        Returns:
+            list:
+        """
+        split_value = re.split(r"(?<!\\):", value)
```

Comment:
When you add escaping, you should also add an option to escape the escape character.